### PR TITLE
Witek/add company value

### PR DESCRIPTION
**Pull request by piowit**

### CODE REVIEW

File: spec/factories/company_values.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :company_value do
+    title { 'MyString' }
```
Comment (ciejj) [inline spec/factories/company_values.rb]: I would consider giving it more fitting name - like _Helpful_ or _Patient_

File: app/views/admin/pages/dashboard.html.erb [hunk 2]
```diff
@@ -1,2 +1,3 @@
 Welcome in dashboard<br />
-<%= link_to 'Admin Kudos', admin_kudos_path %>
+<%= link_to 'Admin Kudos', admin_kudos_path %><br/>
+<%= link_to 'Admin Company Values', admin_company_values_path %>
```
Comment (ciejj) [inline app/views/admin/pages/dashboard.html.erb]: In my understanding, this link should be present in the navbar as well:
_There is a link in admin's navigation to go to company values path_

**Review by ciejj — APPROVED** [review]
Great job 👏

File: app/views/admin/company_values/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @company_values.each do |company_value| %>
+      <tr>
+        <td>
+          <%= link_to 'Edit', edit_admin_company_value_path(company_value), class: 'btn btn-outline-primary btn-sm' %>
+          <%= link_to 'Delete', admin_company_value_path(company_value), method: :delete, class: 'btn btn-outline-danger btn-sm' %>
+          <%= company_value.title %>
```
Comment (oskarlakner) [inline app/views/admin/company_values/index.html.erb:16]: usually title comes first

**Pull request merged**